src/Compilers/CSharp/Test/CommandLine/CommandLineTests.cs

```diff
@@ -4570,6 +4570,28 @@ class C
             CleanupAllGeneratedFiles(file.Path);
         }
 
+        [Fact, WorkItem(1093063, "DevDiv")]
+        public void VerifyDiagnosticSeverityNotLocalized()
+        {
+            string source = @"
+class C
+{
+}
+";
+            var dir = Temp.CreateDirectory();
+
+            var file = dir.CreateFile("a.cs");
+            file.WriteAllText(source);
+
+            var outWriter = new StringWriter(CultureInfo.InvariantCulture);
+            var csc = new MockCSharpCompiler(null, dir.Path, new[] { "/nologo", "/target:exe", "a.cs" });
+            int exitCode = csc.Run(outWriter);
+            Assert.NotEqual(0, exitCode);
+            Assert.Contains("error CS5001:", outWriter.ToString().Trim());
+
+            CleanupAllGeneratedFiles(file.Path);
+        }
+
```

Comment:
I'm not clear on how this test validates that we're not localizing "error". Could you explain?